replit_river/server_transport.py

```diff
@@ -75,6 +87,74 @@ async def handshake_to_get_session(
                 raise InvalidMessageException(error_msg) from e
         raise WebsocketClosedException("No handshake message received")
 
+    async def close(self) -> None:
+        await self._close_all_sessions()
+
+    async def _get_or_create_session(
```

Comment:
we should deprecate this eventually like we did with ts, server should never need this